tests/utils/get_test_data.py

```diff
@@ -20,7 +20,7 @@ def get_test_data():
     Returns:
 
     """
-    _get_test_data_slice(300, 150)
+    _get_test_data_slice(2000, 1500)  # 300, 150
```

Comment:
ME COMPUTR GON DIE -- was there something that broke with the tiny matrix?

Reply:
As we discussed, even on the larger matrix the marker genes process didn't work so wanted to try on a larger dataset. I should not commit this change, will revert to old smaller dimensions!